credentials/alts/internal/handshaker/service/service.go

```diff
@@ -50,7 +52,16 @@ func Dial(hsAddress string) (*grpc.ClientConn, error) {
 		// Disable the service config to avoid unnecessary TXT record lookups that
 		// cause timeouts with some versions of systemd-resolved.
 		var err error
-		hsConn, err = grpc.NewClient(hsAddress, grpc.WithTransportCredentials(insecure.NewCredentials()), grpc.WithDisableServiceConfig())
+		opts := []grpc.DialOption{
+			grpc.WithTransportCredentials(insecure.NewCredentials()),
+			grpc.WithDisableServiceConfig(),
+			grpc.WithKeepaliveParams(keepalive.ClientParameters{
+				Timeout:             5 * time.Second,
```

Comment:
I can't see any other defaults (internally) setting the timeout lower than 10 seconds, so might set this to 10 seconds just to match other defaults.

I'm debating about `Time` to 1 hour. 10 minutes would also be conservative but small enough to be frequently exercised in tests...

cc @ejona86 FYI

Reply:
Ack. I changed the timeout to 10 seconds, and time to 10 minutes. Will check how this changes the behavior we are interested in.